query_service/core/main.py

```diff
@@ -22,6 +22,7 @@
     "http://127.0.0.1:3001",
     "http://3.134.90.242", #aws ip
     "http://18.119.65.244", #graphdb
+    "https://beta.brainkb.org",
```

Comment:
@tekrajchhetri I'll leave it up to you if you want to also add any other origins

Reply:
@aaronkanzer thank you. Could you update the pull request to merge to `[rapid-release-api-support](https://github.com/sensein/BrainKB/tree/rapid-release-api-support)`. It's the latest update I am working on atm and is not yet merged to main.

Reply:
merging --> ignore my previous comment.